packages/database/src/migrations/20211001060307-LESMISDropOutRateProvinceOverlay-modifies-data.js

```diff
@@ -0,0 +1,311 @@
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const PERMISSION_GROUP = 'LESMIS Public';
+const PROVINCE_OVERLAY_GROUP_ID = '5f2c7ddc61f76a513a0001f3';
+
+const getOverlayNames = (gradeNum, educationLevel, GPI) => {
+  if (gradeNum !== null) {
+    return {
+      name: `Grade ${educationLevel === 'p' ? gradeNum : gradeNum + 5} Dropout Rate ${
+        GPI ? 'GPI' : ''
+      }`,
+      dataElement: [
+        `dor_province_${educationLevel.concat(gradeNum)}_${GPI ? 'f' : 't'}`,
+        `${GPI ? 'dor_province_s1_m' : ''}`,
+      ],
+      reportCode: `LESMIS_grade_${educationLevel === 'p' ? gradeNum : gradeNum + 5}_dropout_rate${
+        GPI ? '_GPI' : ''
+      }_province_map`,
+    };
+  }
+  const educationLevelTranslation = {
+    pe: 'Primary',
+    lse: 'Lower Secondary',
+    use: 'Upper Secondary',
+    p40: 'Primary',
+    ls40: 'Lower Secondary',
+  };
+
+  return {
+    name: `${educationLevelTranslation[educationLevel]} Dropout Rate`,
+    dataElement: [`dor_province_${educationLevel}_t`],
+    reportCode: `LESMIS_${educationLevelTranslation[educationLevel]
+      .toLowerCase()
+      .split(' ')
+      .join('')}_dropout_rate_province_map`,
+  };
+};
+
+const getReport = (reportCode, dataElement) => ({
+  id: generateId(),
+  code: reportCode,
+  config: {
+    fetch: {
+      dataElements: dataElement,
+      aggregations: [
+        {
+          type: 'FINAL_EACH_YEAR',
+          config: {
+            dataSourceEntityType: 'district',
+            aggregationEntityType: 'requested',
+          },
+        },
+      ],
+    },
+    transform: [
+      {
+        transform: 'updateColumns',
+        insert: {
+          organisationUnitCode: '=$organisationUnit',
+          value: '=divide($value, 100)',
+        },
+        exclude: ['organisationUnit', 'dataElement', 'period'],
+      },
+    ],
+  },
+});
+
+const getGpiReport = (reportCode, dataElement) => ({
+  id: generateId(),
+  code: reportCode,
+  config: {
+    fetch: {
+      dataElements: dataElement,
+      aggregations: [
+        {
+          type: 'FINAL_EACH_YEAR',
+          config: {
+            dataSourceEntityType: 'district',
+            aggregationEntityType: 'requested',
+          },
+        },
+      ],
+    },
+    transform: [
+      {
+        transform: 'updateColumns',
+        insert: {
+          organisationUnitCode: '=$organisationUnit',
+          '=$dataElement': '=$value',
+        },
+        exclude: ['dataElement', 'value', 'organisationUnit'],
+      },
+      {
+        transform: 'mergeRows',
+        groupBy: ['organisationUnitCode', 'period'],
+        using: 'single',
+      },
+      {
+        transform: 'updateColumns',
+        insert: {
+          value: `=divide($${dataElement[0]},$${dataElement[1]})`,
+        },
+        exclude: [dataElement[0], dataElement[1]],
+      },
+    ],
+  },
+});
+
+const getTargetDistrictReport = (reportCode, dataElement) => ({
```

Comment:
This isn't actually being called anywhere right? Can't have a a target district report at province level. It's generally preferable to delete unused code because it adds unnecessary confusion

Reply:
Yes, you're right, will remove.